### PR TITLE
2019/accidental: add talk for Golang Adelaide meeting

### PR DESCRIPTION
**Pull request by kortschak**
Please take a look.

### CODE REVIEW

**Comment by kortschak** [comment]
*ping*

File: 2019/accidental.slide [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+Gonum follows the principle that incorrect software in a scientific setting should not continue running
+
+In the world of bioinformatics, incorrect results can lead to significant harm ([[https://www.nytimes.com/2011/07/08/health/research/08genes.html][erroneous drug trial approval for example]])
+
+Otherwise, idiomatic Go code
+
+
+* Notable Gonum package clients
+
+- Gorgonia — machine learning for Go
```
Comment (sbinet) [inline 2019/accidental.slide]: perhaps add links to the projects of this list? (when available)

**Review by sbinet — APPROVED** [review]
LGTM

**Review by btracey — APPROVED** [review]
Nice talk!

File: 2019/accidental.slide (hunk 1 above)
Comment (kortschak) [reply 2019/accidental.slide]: Done

**Comment by kortschak** [comment]
Thanks

**Comment by vladimir-ch** [comment]
Yes, very nice!

**Pull request merged**